frontend/src/routes/train.jsx

```diff
@@ -0,0 +1,94 @@
+import { createEffect, createSignal } from "solid-js";
+import { createStore } from "solid-js/store";
+import { styled } from "solid-styled-components";
+import Button from "../components/form/button";
+import InputEmail from "../components/form/email-input";
+import InputHash from "../components/form/hash-input";
+import Slider from "../components/slider";
+import InputFile from "../components/form/file-input";
+import { getFilesUrls } from "../helpers/getFilesUrl";
+
+const Train = () => {
+  // store of data from inputs of form
+  const [formData, setFormData] = createStore({
+    email: "",
+    files: [],
+    hash: "",
+  });
+
+  // use state images urls
+  const [imagesList, setImagesList] = createSignal([]);
+
+  // set form data when inputs changed
+  const handleFormChange = (e) => {
+    const { target } = e;
+    setFormData(() => {
+      switch (target.type) {
+        case `email` || `text`:
+          return { [target.name]: target.value };
+        case `file`:
+          return { [target.name]: target.files };
+        default:
+          return;
```

Comment:
Please, do not use switch case everywhere

Reply:
done